src/ensembl/src/shared/radio/Radio.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import defaultStyles from './Radio.scss';
+
+export type RadioOption = {
+  value: string | number | boolean;
+  label: string;
+};
+
+export type RadioOptions = RadioOption[];
+
+type Props = {
+  onChange: (selectedOption: string | number | boolean) => void;
+  classNames?: any;
```

Comment:
surely not `any`? Is it a string (then className instead of classNames) or an object of strings (and if it's an object, then how is it used)?

Reply:
Updated the type. It just overrides the default classnames.